web/public/js/directives/cd-expandable-container/cd-expander/style.less

```diff
@@ -1,3 +1,6 @@
 *[cd-expander]{
   cursor: pointer;
+  .cd-expander {
+    outline: none;
```

Comment:
Looks terrible since it's now a block item.
Without being block, the "view more" was clickable but not necessarly responsive to the action